src/ts/components/replaybutton.ts

```diff
@@ -0,0 +1,30 @@
+import { ButtonConfig, Button } from './button';
+import { UIInstanceManager } from '../uimanager';
+import { PlayerAPI } from 'bitmovin-player';
+import { i18n } from '../localization/i18n';
+
+/**
+ * A button to play/replay a video.
+ */
+export class ReplayButton extends Button<ButtonConfig> {
+
+  constructor(config: ButtonConfig = {}) {
+    super(config);
+
+    this.config = this.mergeConfig(config, {
+      cssClass: 'ui-replaybutton',
+      text: i18n.getLocalizer('replay'),
+    }, this.config);
+  }
+
+  configure(player: PlayerAPI, uimanager: UIInstanceManager): void {
+    super.configure(player, uimanager);
+
+    this.onClick.subscribe(() => {
+      if (!player.hasEnded()) {
+        player.seek(0);
+      }
+      player.play('ui');
```

Comment:
so we would call `play` either way? So if we are currently paused and the user hits the button we would play again? 

Just wondering if this was considered from an UX perspective.

Reply:
Fair point, I missed this case indeed.

Reply:
Updated in 5d8b0c47

Reply:
Thanks! 

Just wondering, what happens in a live stream? Should it be possible for this component to even show?

Reply:
Another very good question! I think there are cases where it's useful for live streams as well, but that's a slightly more complex topic, let's save that for a future PR. Button is now hidden during live streams: 0e5bae25